ext/standard/tests/strings/str_word_count.phpt

```diff
@@ -34,9 +54,10 @@ var_dump(str_word_count("'foo'", 2, "'"));
 var_dump(str_word_count("-foo-", 2));
 var_dump(str_word_count("-foo-", 2, "-"));
 
-echo "Done\n";
 ?>
---EXPECTF--
+
+DONE
```

Comment:
Side note: Why are you making this change everywhere? It looks unnecessary.

Reply:
I've seen a bunch of test do that in other extensions and else where so I just decided to mimic it for "consistency", I can stop doing that.

Reply:
I don't think I've ever seen this. Possibly you saw `===DONE===`?

In any case, unless the test output would be empty without it, I don't think there's any point to adding a final "done" in any form.

Reply:
Possible yes, Okay will know this for next time :)